www/classes/Controller/Page/LoggedIn/Schrijfregels.php

```diff
@@ -16,7 +16,8 @@ class Schrijfregels extends LoggedInPage
      */
     public function get_content(): string
     {
-        return ViewRenderer::render_view('page.content.schrijfregels', []);
+        return ViewRenderer::render_view('page.content.schrijfregels', [
+        'schrijfregels' => Configuration::instance()->schrijfregels]);
```

Comment:
```suggestion
        return ViewRenderer::render_view('page.content.schrijfregels', [
                'schrijfregels' => Configuration::instance()->schrijfregels
        ]);
```

zou ik mooier vinden